fpga_interchange/chip_info.py

```diff
@@ -904,7 +904,7 @@ def append_children_bba(self, bba, label_prefix):
 
     def append_bba(self, bba, label_prefix):
         bba.str_id(self.key)
-        bba.str_id(self.value)
+        bba.str_id(str(self.value))
```

Comment:
Just for context, what kind of error/bug did the non-str index generate?

Reply:
a non-string value was tripping this assert, iirc: https://github.com/SymbiFlow/python-fpga-interchange/blob/38affcfeb02f132a43830902631250c369546faf/fpga_interchange/converters.py#L43